libsolidity/interface/OptimiserSettings.h

```diff
@@ -23,12 +23,22 @@
 
 #pragma once
 
+#include <liblangutil/Exceptions.h>
+
 #include <cstddef>
 #include <string>
 
 namespace solidity::frontend
 {
 
+enum class OptimisationLevel
```

Comment:
Maybe call it `Presets`? Level sounds like it cannot be fine-tuned.

Reply:
Done.